test/WebJobs.Script.Tests.Integration/ContainerManagement/LinuxContainerInitializationHostServiceTests.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.WebJobs.Script.Tests;
 using Moq;
+using Moq.Protected;
```

Comment:
rename this file to `AtlasContainerInitializationHostedServiceTests` ?

Reply:
Changed 👍